Client/Frontend/Home/ReaderPanel.swift

```diff
@@ -362,7 +368,7 @@ class ReadingListPanel: UITableViewController, HomePanel, SWTableViewCellDelegat
 
         let readerModeImageView = UIImageView(image: UIImage(named: "ReaderModeCircle"))
         containerView.addSubview(readerModeImageView)
-        readerModeImageView.snp_makeConstraints({ (make) -> Void in
+        readerModeImageView.snp_makeConstraints(closure: { (make) -> Void in
```

Comment:
Nit: we can remove these closure parameter names and leave this as a trailing closure